tomato-timer/src/App.js

```diff
@@ -1,8 +1,49 @@
 import React, { Component } from 'react';
+import Login from './googleAuth.js';
 import logo from './logo.svg';
 import './App.css';
+import * as firebase from 'firebase';
+import {configured} from './firebase';
+
+var provider = new firebase.auth.GoogleAuthProvider();
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loggedin: false,
+      name:"",
+      email:"",
+      id:""
+    }
+    this.handleLogIn = this.handleLogIn.bind(this);
+  }
+
+  handleLogIn(){
+    console.log(this.state.loggedin)
+    if(this.state.loggedin){
+      firebase.auth().signOut().then(() => {
+        this.setState({
+          loggedin: false,
+          name:"",
+          email:"",
+          id:""
+        });
+      }).catch(function(error){
+      })
```

Comment:
You have an unhandled error here, if you are going to catch it you should do something about it. Warn the user, log it for debugging, etc.